lib/version/MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include versioneer.py
```

Comment:
I don't think we need this.

Reply:
The file? Or that line? I just followed the `versioneer` instructions and committed everything that was generated. Also, I didn't include a `setup.py` do you think I am okay without it? It seems fine without it.

Reply:
I mean the file. That's for distribution using pip etc.

Oh yeah, the version should also be set for the python kotekan package (in `python/setup.py`).

Reply:
Can you check that the rfi_receiver also ends up with the correct `__version__` from versioneer?

Reply:
> Can you check that the rfi_receiver also ends up with the correct `__version__` from versioneer?

```
[willis@scarlet rfi_receiver]$ python rfi_receiver.py
2019.12+26.gcfc745ee.dirty
```

Yeah that seems okay.

Reply:
> I mean the file. That's for distribution using pip etc.

Ah I see, I will delete that file then.
 
> Oh yeah, the version should also be set for the python kotekan package (in `python/setup.py`).

How do I pass this `setup.py` to `versioneer` from 'lib/version`?

Reply:
The `setup.py` is for installing the kotekan python package. At the moment it hardcodes a version number, so we should tell it where to get the versioneered version number. Actually, this should be done by versioneer automatically (https://github.com/warner/python-versioneer/blob/master/INSTALL.md#installation). Maybe it had troubles finding the `setup.py`.

Reply:
Is this change going to require that we `pip install versioneer` before building kotekan?   If so we need to update the documentation.  We do already require `python-yaml` so we just need to make sure people install `versioneer` at the same time.

Reply:
The way it works is that versioneer itself is added to the kotekan repository (to prevent what you describe).

Reply:
> The `setup.py` is for installing the kotekan python package. At the moment it hardcodes a version number, so we should tell it where to get the versioneered version number. Actually, this should be done by versioneer automatically (https://github.com/warner/python-versioneer/blob/master/INSTALL.md#installation). Maybe it had troubles finding the `setup.py`.

```
def get_root():
    """Get the project root directory.

    We require that all commands are run from the project root, i.e. the
    directory that contains setup.py, setup.cfg, and versioneer.py .
    """
    root = os.path.realpath(os.path.abspath(os.getcwd()))
    setup_py = os.path.join(root, "setup.py")
    versioneer_py = os.path.join(root, "versioneer.py")
```

According to the documentation `versioneer` needs to be run in the same directory as `setup.py`, `setup.cfg` and `versioneer.py`. So I would need to move `versioneer.py` and `setup.cfg` to `python/`.

Reply:
My latest changes should fix this